azcopy.wxs

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Product Id="*" Name="azcopy" Language="1033" Version="10.26.0" Manufacturer="Microsoft" UpgradeCode="abeee33a-0cca-4060-89d7-0687f7374f7a">
```

Comment:
Will this version need to change every release? If so, we should add a section to the release process to double check this.

Reply:
yes this is the PR in progress not ready for review. Yes will not be using the Version as static value. Will make the changes for it